### PR TITLE
Add IPv6 CMSG options to Fuchsia

### PR DESCRIPTION
**Pull request by brunowonka**

### CODE REVIEW

File: src/fuchsia/mod.rs [hunk 1]
```diff
@@ -974,6 +974,11 @@ s_no_extra_traits! {
         pub sigev_notify_attributes: *mut pthread_attr_t,
         pub __pad: [::c_char; 56 - 3 * 8 /* 8 == sizeof(long) */],
     }
+
+    pub struct in6_pktinfo {
+        pub ipi6_addr: ::in6_addr,
+        pub ipi6_ifindex: ::c_int,
```
Comment (JohnTitor) [inline src/fuchsia/mod.rs:980]: Isn't it `c_uint`?

Comment (brunowonka) [reply src/fuchsia/mod.rs]: you are [correct](https://cs.opensource.google/fuchsia/fuchsia/+/main:zircon/third_party/ulib/musl/include/netinet/in.h;l=308;drc=7c5e521391fddb98fd8f6970da7c410899ddf5cf).

**Review by JohnTitor — APPROVED** [review]

**Comment by JohnTitor** [comment]
Thanks! @bors r+

**Comment by akonradi** [comment]
Looks like the `Debug` impls for these structs are all manual, you probably need to add one here: https://github.com/rust-lang/libc/blob/e0558a931b3d3e757adea4927048d2ced7af3faa/src/fuchsia/mod.rs#L1308

**Comment by brunowonka** [comment]
I didn't realize I had put `in6_pktinfo` in the section of the file that doesn't derive traits automatically. Should be good now.

**Comment by JohnTitor** [comment]
👍, @bors r+

**Pull request merged**